modules/samples/watertank/feature/feature/src/main/resources/agent/water-tank.lua

```diff
@@ -1,24 +1,76 @@
-DHT = require("dht_lib")
-
-dht_data = 2
-buzzer = 1
-gpio.mode(buzzer, gpio.OUTPUT)
+trig = 5 --IO14
```

Comment:
This is a lua script file which is going to upload into a device which has only 56kbits of memory. So I'm not seeing any point to add our license text to this. Since we don't have any other option rather than adding full license header, I kept this file without having our standard license header.